R/dirAliases.R

```diff
@@ -20,65 +20,35 @@ dirOut = function(...) {
 }
 
 # output dir without any subdir.
-dirOutRoot = function(...) {
-	dirWrap("ROUT.DIR", ...)
-}
+dirOutRoot = function(...) ffROUT.DIR(...)
```

Comment:
instead of this, let's just do:

`setff("OutRoot", "ROUT.DIR")`

and then use `ff` instead of `dir` from now on...